aggregator/handler/router/traffic_control_options.go

```diff
@@ -153,128 +141,57 @@ type trafficControlOptions struct {
 // NewTrafficControlOptions creats a new TrafficControlOptions.
 func NewTrafficControlOptions() TrafficControlOptions {
 	return &trafficControlOptions{
-		defaultValue:   defaultDefaultDisabled,
```

Comment:
No initial value for `defaultValue`?

Reply:
It will just be false, can set explicitly